tests/functional/configured_endpoint_urls/test_configured_endpoint_url.py

```diff
@@ -63,6 +66,22 @@ def create_cases():
         )
 
 
+def get_test_case_client_config(client_config):
```

Comment:
Oh I meant changing the parameter name `client_config` to `test_case_client_config` and not the test name.